content/guide/netlify.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Netlify for Quire
+weight: 215
+type: essay
+---
+
+## Netlify 
+
+{{< q-figure id="netlify-logo" >}}
+
+Netlify is a web developer platform that multiplies productivity.
```

Comment:
These three paragraphs sounds like a quote directly from Netlify's site? If so let's put them in as a blockquote.

And let's then add in an unquoted bit about why Netlify for Quire is cool and useful. Maybe something like:

For Quire, Netlify can automatically generate a preview site to share with your collaborators every time you push changes to your project, and can also use it to host your final project when its ready to publish.

Reply:
Ok cool yes they are, sounds good !